virt/learn_more_about_openshift_virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+include::modules/virt-document-attributes.adoc[]
+[id="learn_more_about_openshift_virtualization"]
+= Learn more about {VirtProductName}
+:context: learn_more_about_openshift_virtualization
+
+toc::[]
+
+Use the following sections to find content to help you learn about and use {VirtProductName}.
+
+[id="cluster-administrator"]
+== Cluster Administrator (Maintenance)
+
+[options="header",cols="4*"]
+|===
+|Learn about {VirtProductName} |Plan for {VirtProductName} deployment |Deploy {VirtProductName}|Additional resources
```

Comment:
I think the "OpenShift Virtualization" in each of the headers is redundant.  They already know they are in the "OpenShift Virtualization page". I think you can simplify to "Learn", "Plan", "Deploy" to tighten up the table headers. This applies to all the table on the page.

Reply:
Got rid of all product name variables in the headers